lbr4_base_link__to__table_1_center:
  orientation:
    w: -0.0019237363338867296
    x: 0.005716611023314217
    y: -0.011569683562974897
    z: -0.9999148773866602
  position:
    x: -0.7761920547843166
    y: 0.010998773230037256
    z: 0.1638862777761344
lbr4_base_link__to__table_2_center:
  orientation:
    w: 0.0018803143383833588
    x: 0.015729181025960368
    y: 0.0041155186082523845
    z: -0.9998660509232355
  position:
    x: -0.8758429227641744
    y: 0.022486135743135766
    z: 0.016820892558890743
num_thetas: 8
num_xs: 10
num_ys: 15
table_type: Medium (36in X 24in)
table_x: 0.61
table_y: 0.915
table_z: 0.595
theta_max: 3.14159265359
theta_min: -3.14159265359
x_max: 0.275
x_min: -0.275
y_max: 0.425
y_min: -0.425
z: 0
